controller/volume_controller.go

```diff
@@ -1617,7 +1621,8 @@ func (vc *VolumeController) reconcileVolumeSize(v *longhorn.Volume, e *longhorn.
 	return nil
 }
 
-func (vc *VolumeController) checkForAutoAttachment(v *longhorn.Volume, e *longhorn.Engine, rs map[string]*longhorn.Replica, allScheduled bool) error {
+func (vc *VolumeController) checkForAutoAttachment(v *longhorn.Volume,
```

Comment:
Line breaks? Not sure if it's necessary.

Same below.

Reply:
Sure, just changed them back. Thanks.